impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -188,9 +186,7 @@ static constexpr vk::ImageUsageFlags ToVKImageUsageFlags(
     } else {
       vk_usage |= vk::ImageUsageFlagBits::eColorAttachment;
     }
-    if (supports_framebuffer_fetch) {
-      vk_usage |= vk::ImageUsageFlagBits::eInputAttachment;
-    }
+    vk_usage |= vk::ImageUsageFlagBits::eInputAttachment;
```

Comment:
You can use subpassLoad on any image. If a depth-stencil image is set in the descriptor, that will work too as long as there is sufficient synchronization. So, no. Though I just removed this code since supports_framebuffer_fetch will always be true in this code.